### DISCUSSION TITLE
My team org is no longer showing

### DISCUSSION THREAD

**Discussion by philparv** [vercel/community#4093] [Account Issue]
### Question

Hi,

My domain has been kicked offline due to payment, however, I am no longer able to see my team in the dashboard. We had a recurring payment setup, but still not able to see my org anymore. Please escalate issue as this is effect our production environment

**Comment by amyegan** [comment]
Hey @philparv. Are you able to [submit a case](https://vercel.com/help#issues)? I would need to know more about your account and your Team to be able to investigate this problem. We can't safely discuss personal account details here in public, and opening a case will give you a more private way of communicating with our support team.
